ras_party/controllers/account_controller.py

```diff
@@ -168,6 +169,16 @@ def change_respondent_enrolment_status(payload, session):
     for case in get_cases_for_collection_exercise(survey_id, business_id, respondent_id):
         post_case_event(case['id'], business_id, category=category, desc=description)
 
+    session.commit()
+    enrolment = query_enrolment_by_survey_business(business_id, survey_id, session)
+    if not enrolment:
+        logger.info('No active enrolments', business_id=business_id, survey_id=survey_id)
+        for case in get_cases_for_collection_exercise(survey_id, business_id, business_id):
+            post_case_event(case['id'],
+                            party_id=None,
```

Comment:
Should we remove party_id here at some point as it will become redundant without BI plus you already have the case id?

Reply:
You're right when we remove BI cases we'll no longer have to create new cases off the back of old ones and that'll mean we never have to send `party_id` as part of the case event request. Until we've made that change in case service I think we should leave it in here though. 
We still need to send a `party_id` for the `RESPONDENT_ENROLED` category